Add more badges :)

vdemeester opened the pull request:
Build badges are for master, experimental, windows and freebsd. Also added a _version_ badge. 🐙

I know this is a bit silly :wink: 🐸 !

/cc @jfrazelle @thaJeztah 

Signed-off-by: Vincent Demeester vincent@sbr.pm

thaJeztah commented:
Nice! Should we (can we) add a separate badge for the Windows builds as well?

vdemeester commented:
@thaJeztah true :sweat_smile:.

I'm not sure if the message on the badge can be customized though so we would either do :
- a list or a table with the different target and their build status)
- a `Docker *mega* Master` build that *aggregates* all the target build (all Linux(es), windows, freebsd, …)

thaJeztah commented:
@vdemeester not sure; maybe @jfrazelle knows if it's possible to have Jenkins generate/use custom badges for specific builds? Otherwise we could use a table

jfrazelle commented:
You can't customize them

thaJeztah commented:
Thanks Jess, that'll mean a table is probably the cleanest solution, e.g.,

Linux | Windows | GoDocs
--- | --- | ---
badge | badge | badge

GordonTheTurtle commented:
Please sign your commits following these rules:
https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work
The easiest way to do this is to amend the last commit:
~~~console
$ git clone -b "badges-all-the-things" git@github.com:vdemeester/docker.git somewhere
$ cd somewhere
$ git rebase -i HEAD~2
editor opens
change each 'pick' to 'edit'
save the file and quit
$ git commit --amend -s --no-edit
$ git rebase --continue # and repeat the amend for each commit
$ git push -f
~~~

Ammending updates the existing PR. You **DO NOT** need to open a new one.

vdemeester committed:
Add jenkins and godoc badges

Build badges are for master and one for experimental (in the `README.md`
of the experimental folder)

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

vdemeester commented:
Something like this ? :blush:.
 
![2015-08-30-212644_956x419_scrot](https://cloud.githubusercontent.com/assets/6508/9569048/d5a09bb4-4f5d-11e5-97f1-3843b5bf1aa7.png)

![2015-08-30-213216_666x412_scrot](https://cloud.githubusercontent.com/assets/6508/9569073/9ba656a0-4f5e-11e5-9e39-f3dd653aeaa9.png)

thaJeztah commented:
Thanks Vincent! Maybe we should move them more to the top of the document? (But you may want to wait until others have commented on this PR)

vdemeester commented:
@thaJeztah yeah, I'll wait a bit :stuck_out_tongue_closed_eyes:

calavera commented:
I like it where it is now, you don't need to see the build statuses when you start to learn about docker.

LGTM

jfrazelle commented:
me likey good idea LGTM

thaJeztah commented:
\o/ thanks @vdemeester!

vdemeester commented:
:tada:
